src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -253,11 +262,11 @@ handleNewGameMenuEvent scenarioStack@(curMenu :| rest) = \case
   VtyEvent ev -> do
     menu' <- nestEventM' curMenu (handleListEvent ev)
     uiState . uiMenu .= NewGameMenu (menu' :| rest)
-  _ -> continueWithoutRedraw
+  _ -> pure ()
  where
   showLaunchDialog = case snd <$> BL.listSelectedElement curMenu of
     Just (SISingle siPair) -> Brick.zoom (uiState . uiLaunchConfig) $ prepareLaunchDialog siPair
-    _ -> continueWithoutRedraw
+    _ -> pure ()
```

Comment:
`continue` does not exist and is the default behavior of the EventM monad.

Reply:
No, there is still a bit of documentation that refers to `continue`, but ever since brick 1.0 `continue` was removed from the API.  Continuing the loop with a redraw is the default action; we only have to call `continueWithoutRedraw` if we specifically do not want to redraw the screen.